css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,70 @@
+.container {
+  padding: 10px;
+}
+
+.row {
+  text-align: center;
+  padding: 5px;
+  font-family:
+    "DM Sans",
+    sans-serif;
+  src: url("https://fonts.gstatic.com/s/dmsans/v11/rP2Hp2ywxg089UriCZ2IHTWEBlwu8Q.woff2") format("woff2");
+}
+
+.display {
+  display: flex;
+  position: relative;
+  width: 100%;
+  padding-bottom: 10px;
+  padding-left: 10px;
+  padding-right: 10px;
+  justify-content: center;
+}
+
+.col-on-fifth {
```

Comment:
Is this supposed to be `col-one-fifth`? If so, its only styling should be `flex-basis: 20%`